extensions/gg-deals/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "gg-deals",
+  "title": "gg.deals",
+  "description": "Fetch the latest deals from gg.deals",
+  "icon": "icon.png",
+  "author": "Whitespace",
+  "categories": [
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search Games...",
+      "description": "Template for showing a static grid",
```

Comment:
consider changing the description for something more aligned with your extension